schema-registry/rest-service/src/test/java/com/hortonworks/registries/schemaregistry/avro/util/CustomParameterizedBlockJUnit4Runner.java

```diff
@@ -0,0 +1,242 @@
+/**
```

Comment:
@michaelandrepearce We are not modifying any of the glassfish sources, but we are using those classes as libraries in classpath. AFAIK, it comes under "GPL Classpath Exception" and it is safe to use as library here.